src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -18,6 +18,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Newtonsoft.Json.Serialization;
+using Octokit;
```

Comment:
This using should be removed. I don't think we should be referencing any Github namespaces in the azdo client.